cloudbuild.yaml

```diff
@@ -0,0 +1,117 @@
+##
+# Required images
+#
+# In order for this to run you need to deploy the Pixite Android cloud builders.
+#
+# ```bash
+# git clone git@github.com:pixiteapps/android-cloud-build /tmp/android-cloud-build
+# cd /tmp/android-cloud-build
+# gcloud builds submit .
+# ```
+
+steps:
```

Comment:
This is missing the deploy step, right?

Reply:
Yeah, don't have keys and I've been using Cloud KMS so I didn't implement that (think I commented to that effect).

Reply:
Ah yes sorry, missed the comment on the PR. I'll add